zombies.js

```diff
@@ -178,7 +252,20 @@
  * @name eat
  * @param {Food} itemToEat  The food item to eat.
  */
-
+Player.prototype.eat = function(itemToEat) {
+  let chosenFood = this.getPack().indexOf(itemToEat);
+  if(itemToEat instanceof Food && chosenFood !== -1) {
+    if ((itemToEat.energy + this.health) > this.getMaxHealth()) {
+      this.health = this.getMaxHealth();
+      this.discardItem(itemToEat);
```

Comment:
Can you think of a way to only call `this.discardItem(itemToEat)` only once in this function?